Makefile

```
@@ -51,7 +51,7 @@ DOCKER_BUILDX_PUSH            := $(DOCKER_BUILDX) --push
 DOCKER_BUILDX_PUSH_X_PLATFORM := $(DOCKER_BUILDX_PUSH) --platform ${DOCKER_BUILD_PLATFORM}
 
 DOCKER_BUILD_PLATFORM_ALPINE         ?= linux/amd64,linux/arm64,linux/ppc64le,linux/arm/v6,linux/arm/v7
-DOCKER_BUILD_RUNTIME_IMAGE_ALPINE    ?= alpine:3.20.0
+DOCKER_BUILD_RUNTIME_IMAGE_ALPINE    ?= alpine:3.20.1
```

```suggestion
DOCKER_BUILD_RUNTIME_IMAGE_ALPINE    ?= alpine:3
```

@JoelSpeed what do you think about changing it to `alpine:3` instead of using a specific minor and patch version?

---

@kvanzuijlen what's your opinion on this? I would like to use the higher order tag as it would be beneficial to always fetch the latest version of alpine. I don't think we should be this specific concerning the base image

---

@tuunit I'd rather auto-merge this PR. Imo builds should be reproducible, but patch/minor level changes to such dependencies shouldn't bother us